docs/.vuepress/config.js

```diff
@@ -1,6 +1,7 @@
 const path = require('path')
 
 module.exports = {
+  port: 8082,
```

Comment:
Shouldn't need to adjust this?

Reply:
I did it because I find myself needing to run one of our other applications at the same time after I've already started the docs. So then I have to stop and restart because the other apps expect to be on `:8080`. I was thinking of instead using a 3000 something port which we never use. I think there is no harm in defining a port to run on

Reply:
vuepress automatically detects when a port is in use. For example, I run apps on :8080 and then vuepress will start iterating from 8080 until it finds a port using `portfinder` package https://github.com/vuejs/vuepress/blob/7e9d0c1f42196dee52daabfaf796e0cdaa7bd6eb/packages/%40vuepress/core/lib/node/dev/index.js#L290

Reply:
I understand but when you run docs on 8080 and then start manager to link a component or something manager doesn't run because it expects 8080. So if docs NEVER run on 8080 we never have to worry about it

Reply:
Yea if it helps your workflow, it's not hurting anyone else!